adapters/mqtt-vertx/pom.xml

```diff
@@ -126,6 +172,22 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
```

Comment:
this plugin seems to expect yaml files being present in `${project.basedir}/src/main/fabric8`, right?